common/lib/xmodule/xmodule/seq_module.py

```diff
@@ -45,6 +45,16 @@ class SequenceFields(object):
         scope=Scope.user_state,
     )
 
+    # Entrance Exam flag -- see cms/contentstore/views/entrance_exam.py for usage
+    is_entrance_exam = Boolean(
+        display_name=_("Is Entrance Exam"),
+        help=_(
+            "Tag this course module as an Entrance Exam.  " +
+            "Note, you must enable Entrance Exams for this course setting to take effect."
+        ),
+        scope=Scope.settings,
+    )
+
```

Comment:
We had briefly discussed making this being "inheritable metadata" to optimize the performance around determine if a setting of a score on a problem is within an entrance_exam, and - thus - need to compute if a milestone fulfillment has been achieved.

Maybe "in_entrance_exam" might be a slightly better name if we decide to use metadata inheritance.